lib/devfest_registration_portal_web/controllers/attendees_controller.ex

```diff
@@ -0,0 +1,25 @@
+defmodule DevfestRegistrationPortalWeb.AttendeeController do
+  use DevfestRegistrationPortalWeb, :controller
+
+  alias DevfestRegistrationPortal.Accounts
+  alias DevfestRegistrationPortal.Accounts.Attendee
+
+  def new(conn, _params) do
+    changeset = Attendee.changeset(%Attendee{}, %{})
+    render(conn, "new.html", changeset: changeset)
+  end
+
+  def create(conn, %{"attendee" => attendee_params}) do
+    case Accounts.register_attendee(attendee_params) do
+      {:ok, _attendee} ->
+        conn
+        |> put_flash(:info, "Welcome #{attendee_params["first_name"]}")
```

Comment:
I would have used the "attendee" in the response tuple instead since its from the repo insertion, its kinda sure. But still, :+1: